man/systemd.network.xml

```diff
@@ -2626,19 +2626,35 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>NextServer=</varname></term>
+        <term><varname>BootServerAddress=</varname></term>
 
         <listitem>
-          <para>Takes an IPv4 address. Configures the next server used by e.g. PXE boot systems.
+          <para>Takes an IPv4 address of the boot server used by e.g. PXE boot systems. When specified,
+          the address is set to the <literal>siaddr</literal> field of the DHCP message header. See
+          <ulink url="https://www.rfc-editor.org/rfc/rfc2131.html">RFC 2131</ulink> for more details.
           Defaults to unset.</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>Filename=</varname></term>
+        <term><varname>BootServerName=</varname></term>
 
         <listitem>
-          <para>Takes a path or url to a file loaded by e.g. a PXE boot loader. Defaults to unset.</para>
+          <para>Takes a name of the boot server used by e.g. PXE boot systems. When specified, the
+          server name is set to option 66. See
+          <ulink url="https://www.rfc-editor.org/rfc/rfc2132.html">RFC 2132</ulink> for more details.
+          Defaults to unset.</para>
```

Comment:
maybe add a comment that typically one of BootServerName=/BootServerAddress= is sufficient to be set, but both can be set too, if desired.